### ISSUE TITLE
Recreating the results of available models

### ISSUE THREAD

**Issue by sophia-wright-blue** [qfgaohao/pytorch-ssd#18]
Hello, I tried to retrain the model for the class names "Handgun,Shotgun" following your instructions:

```
python train_ssd.py --dataset_type open_images --datasets ~/data/open_images --net mb1-ssd --pretrained_ssd models/mobilenet-v1-ssd-mp-0_675.pth --scheduler cosine --lr 0.01 --t_max 100 --validation_epochs 5 --num_epochs 100 --base_net_lr 0.001  --batch_size 5
```

After retraining, I'm not able to get the same object detection results as with your model "gun_model_2.21.pth".

could you give the details of how you trained your model "gun_model_2.21.pth"? were any of the above parameters different? 

thanks,

**Comment by qfgaohao** [comment]
@sophia-wright-blue I don't think I tuned other parameters. Btw, OpenImages dataset is extremely challenging, as it is not as clean as other datasets such as Pascal VOC and COCO. I wouldn't expect a high accuracy either.

**Comment by sophia-wright-blue** [comment]
thanks for replying @qfgaohao , what surprises me is that I'm not able to replicate the results of your model "gun_model_2.21.pth".

a couple of quick follow-up questions:

 how many epochs did you train the model "gun_model_2.21.pth" for?

do you think, reducing the learning rate from 0.01 to 0.001 might help?

as always, thanks for your help,

**Comment by qfgaohao** [comment]
It may be helpful as the Gun dataset is very small. IMHO, more quality data will be more helpful.

**Comment by sophia-wright-blue** [comment]
thanks for replying @qfgaohao , i'll try with a lower learning rate,
